stdlib/date/hour_duration_test.flux

```diff
@@ -29,9 +37,31 @@ outData = "
 ,,0,2018-01-01T00:00:00Z,2030-01-01T00:00:00Z,FF,_m,2018-05-22T19:04:00.538816341Z,0
 ,,0,2018-01-01T00:00:00Z,2030-01-01T00:00:00Z,FF,_m,2018-05-22T19:05:00.676423456Z,0
 ,,0,2018-01-01T00:00:00Z,2030-01-01T00:00:00Z,FF,_m,2018-05-22T19:06:00.982342357Z,0
-"
-t_duration_hour = (table=<-) => table
-    |> range(start: 2018-01-01T00:00:00Z)
-    |> map(fn: (r) => ({r with _value: date.hour(t: duration(v: r._value))}))
+",
+    )
+
+    testing.diff(got: got, want: want)
+}
+
+testcase hour_duration_location {
+    got = csv.from(csv: inData)
+        |> range(start: 2018-01-01T00:00:00Z)
+        |> map(fn: (r) => ({r with _value: date.hour(t: duration(v: r._value), location: {zone: "America/Los_Angeles", offset: 0h})}))
```

Comment:
We shouldn't have any tests that directly call this with `{zone, offset}`. Please use the functions in the `timezone` package as that's the intended use. If someone reads the tests, we don't want them attempting to use the internal representation directly.

You can also likely use `option location = timezone.location(name: "America/Los_Angeles")` as part of the test. I think that as long as it is in the testcase block you can use it in different locations. I am not completely certain of that though.